osquery/events/linux/auditeventpublisher.cpp

```diff
@@ -129,13 +134,122 @@ void AuditEventPublisher::ProcessEvents(
 
       // SELinux events
     } else if (selinux_event_set.find(audit_event_record.type) !=
-               selinux_event_set.end()) {
+                   selinux_event_set.end() &&
+               audit_event_record.fields.find("apparmor") ==
```

Comment:
It's just because separate event codes for AppArmor haven't been merged to Linux kernel upstream yet and all AppArmor events use type=1400, which is SELinux's `AUDIT_AVC` at the same time. That's why the only way to tell AppArmor messages from SELinux ones is to filter them out using `apparmor=` substring. 

This is also the reason why all the messages from AppArmor appear in `selinux_events` table at the moment.

I guess declaring `apparmor=` as a constant will be ok in this case.